tests/components/abode/test_init.py

```diff
@@ -29,6 +30,38 @@ async def test_change_settings(hass):
         await hass.async_block_till_done()
         mock_set_setting.assert_called_once()
 
+    with patch(
+        "abodepy.Abode.set_setting", side_effect=AbodeException((13, "invalid setting"))
+    ):
+        await hass.services.async_call(
+            ABODE_DOMAIN,
+            SERVICE_SETTINGS,
+            {"setting": "confirm_snd", "value": "loud"},
+            blocking=True,
+        )
+
+        assert AbodeException
+
+
+async def test_add_unique_id(hass):
+    """Test unique_id is set to Abode username."""
+    mock_entry = await setup_platform(hass, ALARM_DOMAIN)
+    # Set unique_id to None to match previous config entries
+    hass.config_entries.async_update_entry(entry=mock_entry, unique_id=None)
+    await hass.async_block_till_done()
+
+    config_entry = hass.config_entries.async_get_entry(mock_entry.entry_id)
```

Comment:
We already have the (mock) entry.

Reply:
Ah crap, good point. I was thinking referencing `mock_entry` again would just pull the original config entry data but.... that obviously makes no sense since `mock_entry` is just pointing to the config entry object.